go/test/endtoend/vreplication/resharding_workflows_v2_test.go

```diff
@@ -785,10 +786,10 @@ func createAdditionalCustomerShards(t *testing.T, shards string) {
 	for _, shardName := range arrTargetShardNames {
 		err := cluster.WaitForHealthyShard(vc.VtctldClient, ksName, shardName)
 		require.NoError(t, err)
-		if err := vtgate.WaitForStatusOfTabletInShard(fmt.Sprintf("%s.%s.replica", ksName, shardName), 2); err != nil {
+		if err := vtgate.WaitForStatusOfTabletInShard(fmt.Sprintf("%s.%s.replica", ksName, shardName), 2, 30*time2.Second); err != nil {
```

Comment:
nitpick: wut? why do the `err != nil` check and then on the inside of the `if` we use `require.NoError(t, err)`?

I know you didn't write this, but you forced me to look at it ;)

Reply:
Fixed via 013d4047e5